brownie/network/rpc.py

```diff
@@ -372,3 +387,38 @@ def _check_connections(proc: psutil.Process, laddr: Tuple) -> bool:
         return laddr in [i.laddr for i in proc.connections()]
     except psutil.AccessDenied:
         return False
+
+
+def _validate_cmd_settings(cmd_settings: dict) -> dict:
+    CMD_TYPES = {
+        "port": int,
+        "gas_limit": int,
+        "block_time": int,
+        "time": datetime.datetime,
+        "accounts": int,
+        "evm_version": str,
+        "mnemonic": str,
+        "account_keys_path": str,
+        "fork": str,
+    }
+    for cmd, value in cmd_settings.items():
+        if (
+            cmd in CLI_FLAGS.keys()
+            and cmd in CMD_TYPES.keys()
+            and not type(value) == CMD_TYPES[cmd]
+        ):
+            raise ValueError(
+                f'Wrong type for cmd_settings "{cmd}" ({value}). '
+                f"Found {type(value)}, but expected {CMD_TYPES[cmd]}."
```

Comment:
when outputting a type name, the error string is easier to read if you use the  `__name__` member instead of the `repr` for the type itself.
```suggestion
                f"Found {type(value).__name__}, but expected {CMD_TYPES[cmd].__name__}."
```

Reply:
Ah I was looking for something like this, thx!